d2l-localize-behavior.html

```diff
@@ -47,6 +48,38 @@
 				this._observer.disconnect();
 			}
 		},
+		formatDateTime: function(val, opts) {
+			var formatter = new d2lIntl.DateTimeFormat(this.language, opts);
+			return formatter.format(val);
+		},
+		formatDate: function(val, opts) {
+			var formatter = new d2lIntl.DateTimeFormat(this.language, opts);
+			return formatter.formatDate(val);
+		},
+		formatFileSize: function(val) {
+			var formatter = new d2lIntl.FileSizeFormat(this.language);
+			return formatter.format(val);
+		},
+		formatNumber: function(val, opts) {
+			var formatter = new d2lIntl.NumberFormat(this.language, opts);
+			return formatter.format(val);
+		},
+		formatTime: function(val, opts) {
+			var formatter = new d2lIntl.DateTimeFormat(this.language, opts);
+			return formatter.formatTime(val);
+		},
+		parseDate: function(val) {
```

Comment:
Huh, for formatting there datetime/date/time, but only time/date for parse? Does either of these work with a full datetime or is there just too much variance in datetime representations to bother?

Reply:
They do not, no. For the formatter, `intl` just follows in lock-step with the standard [Intl API](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat) (or as close as possible). There isn't an equivalent parser API though (other than what's in `new Date()`) so we kind of just went with what we'd always had.

I think it makes sense though... typically parsing is done by a separate date input and time input UI component.